src/modules/headset/headset_simulator.c

```diff
@@ -368,6 +368,8 @@ static Pass* simulator_getPass(void) {
 
   uint32_t width, height;
   simulator_getDisplayDimensions(&width, &height);
+  width *= state.config.supersample;
+  height *= state.config.supersample;
```

Comment:
I think `simulator_getDisplayDimensions` should factor in the supersample instead of here, so that `lovr.headset.getDisplayDimensions` returns the supersampled size (like OpenXR does).

Reply:
I considered that one, but it's also used by `simulator_getViewAngles` which would return wrong values if the change was made there.

Reply:
Wait that's not true at all :smile: 

I've pushed the improvement.